modules/cnv-about-fencing.adoc

```diff
@@ -0,0 +1,35 @@
+// Module included in the following assemblies:
+//
+// * cnv/cnv_users_guide/cnv-fencing-nodes.adoc
+
+[id="cnv-about-fencing_{context}"]
+
+= About fencing nodes
+
+Fencing is the act of protecting shared resources in order to safely recover 
+from node failure.
```

Comment:
Perhaps:  Fencing is the act of putting a node into a state where it can be known not to be accessing shared resources,   in order to make use of them elsewhere after a node level failure.